memory/numa_test.py

```diff
@@ -98,6 +99,32 @@ def test(self):
         elif ret != 0:
             self.fail('Please check the logs for failure')
 
+    def test_softoffline(self):
+        """
+        Test PFN's before and after offlining
+        """
+        self.nr_pages = self.params.get(
+            'nr_pages', default=50)
+        os.chdir(self.teststmpdir)
+        self.log.info("Starting test...")
+        cmd = './softoffline -m %s -n %s' % (self.map_type, self.nr_pages)
+        ret = process.system(cmd, shell=True, sudo=True, ignore_status=True)
+        if ret != 0:
+            self.fail('Please check the logs for failure')
```

Comment:
having test case name listed in self.fail messge will be good idea

Reply:
Anyhow test will run in individual manner with respective failures, Is it enough or do we need to add testcase name?

Reply:
i think if we have test case name added should be easy to debug but i fine with this approach